app/views/status_updates/_status_update.html.erb

```diff
@@ -0,0 +1,5 @@
+
+
+<p><%=User.find(status_update.user_id).name%> at <%=status_update.created_at%></p>
```

Comment:
Don't do queries in views - if the status update belongs to a user you can just do status_update.user.name

or in the status_update model declare `delegate :name, to: :user, prefix: true`
That will generate a user_name instance method on your StatusUpdate.